temporian/implementation/numpy_cc/operators/calendar.cc

```diff
@@ -0,0 +1,258 @@
+#include <pybind11/numpy.h>
+#include <pybind11/pybind11.h>
+
+#include "absl/time/time.h"
+
+#include "temporian/implementation/numpy_cc/operators/common.h"
+
+namespace {
+namespace py = pybind11;
+
+static absl::TimeZone parse_tz(const py::object tz) {
```

Comment:
Can you comment on the pro/con of using a reference?
const py::object tz
vs
const py::object& tz

Reply:
I changed this implementation and now it will receive a reference because of a change in design however this discussion still applies to `map_week_day`.

I left the weekday as a const reference to indicate that I don't want the function to modify the weekday but I don't want to make create unnecessary copies of the value. Is this reasoning correct? is there another way to indicate this? does it even make sense to consider this given that the weekday is just an enum so copying the value should not be a problem?

Sorry to respond your question with 3 new questions :)

Reply:
>> I changed this implementation and now it will receive a reference because of a change in design however this discussion still applies to map_week_day.

I still see no reference. Are you sure you pushed your change?

What about the following conversion:
- const& => Input argument that cannot be modified.
- const => A small input argument that cannot be modified.
- * (pointer) => An argument that is expected to be modified.

Reply:
Done